index.html

```diff
@@ -309,11 +314,19 @@ <h3>
 
           <ol>
             <li>
+Remove the <code>did:web:</code> prefix.
+            </li>
+            <li>
 Replace ":" with "/" in the method specific identifier to obtain the fully
 qualified domain name and optional path.
             </li>
             <li>
-If the domain contains a port percent decode the colon.
+If the domain contains a port, percent decode the colon.
+            </li>
+            <li>
+Percent-decode each path fragment. (Note that although URI paths are technically
+allowed to contain <code>:</code> characters, doing so is not recommended, and
+may hinder interoperability.)
```

Comment:
Wouldn't percent-decoding each path part mean that percent-encoded non-URL-safe characters in the DID would end up not being representable in the HTTPS URL (because they need to be percent-encoded)? These would then have to be double-percent-encoded in the DID.

e.g.
`did:example:example.org:dir:space%20bar` → `https://example.org/dir/space bar.json`
`did:example:example.org:dir:space%25%20bar` → `https://example.org/dir/space%20bar.json`
while I would think the desired case would be:
`did:example:example.org:dir:space%20bar` → `https://example.org/dir/space%20bar.json`

Would it make more sense to just say percent-decode the colon character in each path part, similar to how that is done for the domain part in the previous step? i.e. replace occurrences of `%3A` with `:` in each path part.

Maybe saying "path part" instead of "path fragment" could be more clear, so that the term [fragment](https://www.w3.org/TR/did-core/#fragment) could be used more consistently?